lib/maintainers/runner.rb

```diff
@@ -183,13 +183,12 @@ def report_lists_details(maintainers_files)
         csv << ["repo","repo_group","maintainer"]
         maintainers_files.keys.sort.each { |repo|
           maintainers = JSON.load( maintainers_files[repo] )
-          list = maintainers['internal_list']
-          if list 
-            maintainers['people'].each { |person|
-              name =  "#{person['email'] ? person['email'] : person['name'] ? person['name'] : person['github']}"
-              csv << [repo,list, name ]
-            }
-          end
+          list = "#{maintainers['internal_list']}"
+          maintainers['people'].each { |person|
+            name =  "#{person['email'] ? person['email'] : person['name'] ? person['name'] : person['github']}"
+            github = "#{person['github']}"
+            csv << [repo, list, name, github]
```

Comment:
This should make it easier for IT to manage github teams - my only concern is with  line 183 - I think @bcall needs a header title...